### PR TITLE
Adding the validation to run only in qa environments

### PR DESCRIPTION
**Pull request by didierllbean**

### CODE REVIEW

**Comment by jrdavisllbean** [comment]
Hi @didierllbean, when I switched to your ctauto_731 branch, I noticed there was a merge conflict in ODSEventPage.java. See ctauto731.jpg for screen shot. Note I have pulled the latest build from master and this is still happening.

ctauto731

**Comment by jrdavisllbean** [comment]
When I rebased with master, the merge conflict was resolved. I will continue code reviewing.

**Comment by jrdavisllbean** [comment]
@didierllbean, I am not seeing ecwebq11 in config.properties or isTestingEnvironment method? do we ever use qa11? If we do please add as a property in config.properties, condition in isTestingEnvironment. If we do run automation against qa11 then please ignore this request.

**Comment by srtester** [comment]
Currently we do not sue Q11 for general QA, it is primarily used for load testing.

**Comment by jrdavisllbean** [comment]
Hi @didierllbean, when LLBeanServer uses the https protocol, the "not a valid test URL." mesage appears. When http is used, the test executes without problem. do we want to account for https as well as http? 

When I changed BaseTests line 235 to url = url.replaceAll("http[s]*:","").replace("/","");, both http and https work fine and the test executed.

**Will cause exception**
        <parameter name="LLBeanServer" value="https://m-ecwebq.llbean.com/"/>

**Will run test**
        <parameter name="LLBeanServer" value="http://m-ecwebq.llbean.com/"/>

If we are not going to use https for LLBeanServer then this pull request is all set. I have confirmed when LLBeanServer contains either www.llbean.com or m.llbean.com, then the message "https://www.llbean.com/ is not a valid test URL. Please used the authorized QA site to run the automation and  please do not use production URL's when running the automation suite." appears.

The other non-prod URL's, m-ecwebq.llbean.com, m-ecwebs01.llbean.com, ecwebq12.llbean.com, ecwebq13.llbean.com and ecwebs01.llbean.com are working as intended.

File: src/main/java/com/llbean/automation/web/tests/BaseTests.java [hunk 1]
```diff
@@ -230,4 +231,17 @@ private void handleABTesting(String testSegment) throws ExceptionHandler {
         } else
             throw new ExceptionHandler(llBeanTexts.configProperties.getProperty("no_A_BTestingCorrectValueFound") + testSegment);
     }
+
+    private boolean isTestingEnvironment(String url){
+        url = url.replace("http[s]*","").replace("/","");
+
```
Comment (jrdavisllbean) [inline src/main/java/com/llbean/automation/web/tests/BaseTests.java]: Hi @didierllbean, I am still not able to execute the tests using https. Please change the first replace to replaceAll then add a : after the *.

This code should work fine: url = url.replaceAll("http[s]*:","").replace("/","");

**Pull request merged**